source/overloading.tex

```diff
@@ -4100,7 +4099,8 @@
 A \defnx{string literal operator template}{literal!operator!template string}
 is a literal operator template whose \grammarterm{template-parameter-list}
 comprises
-a single non-type \grammarterm{template-parameter} of class type.
+a single \grammarterm{parameter-declaration} that declares a
```

Comment:
Isn't a `parameter-declaration` a grammar production for function parameters? That doesn't seem to fit here, it can't declare an NTTP..

Reply:
I disagree.   See the grammar in [temp.param].

Reply:
I think I understand what this is trying to say now. It's a little awkward that this doesn't give enough context that this is talking about a `\grammarterm{template-parameter}` produced from a single `\grammarterm{parameter-declaration}`. What do you think about phrasing that something along these lines?

".. is a literal operator template whose \grammarterm{template-parameter-list}
comprises a single \grammarterm{template-parameter} produced from a \grammarterm{parameter-declaration}, which declares a ..."

Reply:
I don't think we use "produce" in these contexts.  And when you look at [temp.param] and  [temp.pre] and the meaning of "comprises", I can't see how you can misunderstand the sentence here.  Please re-read the text with the a-priori understanding that a _parameter-declaration_ can naturally appear in a _template-parameter-list_, when declaring a non-type template parameter.

The standard is not a tutorial.

Reply:
Still, as an implementor, I was a bit confused here.

Another suggestion, using similar-to-existing wording:

".. is a literal operator template whose \grammarterm{template-parameter-list}
comprises a single \grammarterm{parameter-declaration} of a \grammarterm{template-parameter}, which declares a ..."